### PR TITLE
Multifield email

### PR DESCRIPTION
**Pull request by heron2014**
## Ready

@nelsonic can you please have a look at this PR, thank you.

Related to [529](https://github.com/FAC-GM/app/issues/529), [531](https://github.com/FAC-GM/app/issues/531#issuecomment-229777253)

Change the mapping on email by creating new property original which is not going to be analyzed by ES, which will improve matching email from csv files.

### CODE REVIEW

**Comment by nelsonic** [comment]
@heron2014 the PR description is _conflicting_ ...  "_Not ready yet_" `||` "_please have a look_" ... ❓

File: test/fixtures/initialize.js [hunk 1]
```diff
@@ -76,6 +85,9 @@ client.indices.exists({index: 'gmcontact'}, function (err, res) {
 
         client.indices.create({index: 'gmcontact'}, function (res, err) {
             client.indices.putMapping({index:"gmcontact", type:"contacts", body:params}, function (err,resp) {
+              console.log('##################');
```
Comment (nelsonic) [inline test/fixtures/initialize.js]: if you need a way of tracing these errors in the Heroku logs, you might want to use a more distinctive string... 😉

**Comment by nelsonic** [comment]
@heron2014 / @SimonLab code looks good. please update PR description and _confirm_ when PO approves ping me on Gittererer. 👍

File: test/fixtures/initialize.js (hunk 1 above)
Comment (heron2014) [inline test/fixtures/initialize.js]: I have deleted these unnecessary console in the following PR.

**Comment by heron2014** [comment]
@nelsonic PO approved this feature. :+1:

**Pull request merged**